.github/workflows/build-images.yml

```
@@ -0,0 +1,77 @@
+name: build-images
+
+on:
+  push:
+    branches:
+      - edge
+  release:
+  workflow_dispatch:
+
```

Restrict (or add) permissions to the job. When the ghcr is used the permission to write to the packages is needed. Always specify as less as possible here. The restricted token is also the default for new repos so it only works without in older repos. See [token permission documentation](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token)
```suggestion

permissions:
  contents: read
  packages: write

```

---

the token is not used add the moment. But a good addition

---

Maybe add the `packages: write` commented out like the ghcr login part is currently then?